core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -288,12 +305,13 @@ public FileInfo getFileInfo(String fileId) throws IOException {
         .setInMemoryPercentage(cachedPercentage);
   }
 
-  private alluxio.grpc.FileInfo buildFileInfoFromUfsStatus(UfsStatus status,
-      String filename, String alluxioFilePath, String ufsFullPath) {
+  private DoraMeta.FileStatus buildFileStatusFromUfsStatus(UfsStatus status, String ufsFullPath) {
+    String filename = new AlluxioURI(ufsFullPath).getName();
+
     alluxio.grpc.FileInfo.Builder infoBuilder = alluxio.grpc.FileInfo.newBuilder()
         .setFileId(ufsFullPath.hashCode())
         .setName(filename)
-        .setPath(alluxioFilePath)
+        .setPath(ufsFullPath)
```

Comment:
why is this change?

Reply:
In the original code, the `alluxioFilePath` is the same as `ufsFullPath`, because now Dora client pass full UFS path to Worker.

So, in the code refactoring, I remove the param `alluxioFilePath` and used the same `ufsFullPath` for setPath() and setUfsPath().